compiler/powershell.vim

```diff
@@ -0,0 +1,63 @@
+" Compiler:	powershell
+" Run ps1 scripts in powershell and process their output. Quickly jump through
+" stack traces and see script output in the quickfix.
+
+if exists("current_compiler")
+  finish
+endif
+let current_compiler = "powershell"
+
+if exists(":CompilerSet") != 2		" older Vim always used :setlocal
+  command -nargs=* CompilerSet setlocal <args>
+endif
+
+let s:cpo_save = &cpo
+set cpo-=C
+
+" Show CategoryInfo, FullyQualifiedErrorId, etc?
+let g:ps1_efm_show_error_categories = get(g:, 'ps1_efm_show_error_categories', 0)
+
+" Use absolute path because powershell requires explicit relative paths
+" (./file.ps1 is okay, but # expands to file.ps1)
+CompilerSet makeprg=powershell\ %:p
```

Comment:
powershell is Windows-only. I recommend some way to configure this - or even default it to be - pwsh (cross-platform) or powershell.

Reply:
added g:ps1_makeprg_cmd